src/auto/functions.rs

```diff
@@ -40,13 +41,17 @@ pub fn ascii_strcasecmp(s1: &str, s2: &str) -> i32 {
     }
 }
 
-//pub fn ascii_strdown(str: &str, len: /*Unimplemented*/Fundamental: SSize) -> Option<String> {
-//    unsafe { TODO: call ffi::g_ascii_strdown() }
-//}
+pub fn ascii_strdown(str: &str, len: isize) -> Option<String> {
+    unsafe {
```

Comment:
This function is not safe as-is, but with my array-length change it will be.

Best to ignore all such functions for now